src/components/dialogs/FetchErrorDialog.vue

```diff
@@ -0,0 +1,52 @@
+<template>
```

Comment:
Very similar to dialog in Filings UI except I changed "filing" to "application".